src/Angor/Client/Pages/Invest.razor

```diff
@@ -262,6 +262,12 @@
             else
             {
                 project = SessionStorage.GetProjectById(ProjectId);
+
+                if (project == null)
+                {
+                    NavigationManager.NavigateTo($"/view/{ProjectId}");
+                    return;
```

Comment:
Is return required after navigate to?

Reply:
yes otherwise execution continue and we get an exception bellow